src/components/Charts/Charts.style.ts

```diff
@@ -18,10 +11,14 @@ const charts = {
 
 const tooltip = {
   width: '200px',
-  boxShadow: `3px 3px 5px ${palette.chart.tooltip.shadow}`,
   fontSizeTitle: '11px',
 };
 
+/** Gets the chart palette based on the current theme. */
+function palette(props: any) {
+  return props.theme.palette.chart;
+}
```

Comment:
This is the core change.  Instead of accessing `palette` directly, we always access it via `props.theme.palette` so that it picks up the current theme.  The rest of the changes are just changing to use this helper function.

See [here](https://github.com/covid-projections/covid-projections/pull/825/files#diff-f5680937bf6b070548b0898693f6fc04R69) for how I override the theme for the share charts.